Regular Expressions/RegExp.sublime-syntax

```diff
@@ -25,7 +25,7 @@ contexts:
     - include: escaped_char
     - include: charset
     - include: group
-    - match: '([?*+][?+]?)|\{\d*,?\d*\}'
+    - match: '([?*+][?+]?)|\{\d+(,\d*)?\}'
```

Comment:
Sorry for the confusion. What I want to say is it would get a False since `a{0}` can match a empty string in the string `b`.